criteria/open-licenses.md

```diff
@@ -7,7 +7,7 @@ order: 12
 ## Requirements
 
 * All code and documentation MUST be published under a license that allows it to be freely reusable, changeable and redistributable
-* Software source code this MUST be licensed under an [OSI-approved open source license](https://opensource.org/licenses/category)
+* Software source code MUST be licensed under an [OSI-approved open source license](https://opensource.org/licenses/category)
 * All code MUST be published with a license
```

Comment:
This appears to be the same as the first bullet, just less specific. If so, can we delete this? If not, can we make this more clear?

Reply:
If it's meaningfully different from the first bullet, than I suggest moving this to the top of the list so that it goes from broader to narrower as you go down.

Reply:
Great idea!

Reply:
Which comment are you replying to?

Reply:
The second one :)